content/docs/start/visualization.md

```diff
@@ -0,0 +1,21 @@
+# Visualization with DVC Plots
+
+In parallel with versioning parameters and metrics by associating them with
+experiments, DVC allows to generate plots from multivariate outputs. There are
+several options to generate and keep track of the plots:
+
+- DVC can generate HTML files that includes interactive Vega-Lite plots, from
+  the data series in JSON, YAML, CSV, or TSV.
+
+- DVC can keep track of image files produced as plot outputs from the
+  training/evaluation scripts.
+
+- DVC, with its DVCLive integration can produce plots automatically during
+  training.
+
+In this document, we'll focus on the first option and briefly describe the other
+two. Please see `dvc plots` and the DVCLive documentation for details.
+
+- [ ] Configure the plots to use dvclive output
```

Comment:
I'm concerned a bit that `dvclive` is just a specific type of a plot and we start with it here. Does it make sense to start with something like confusion matrix here instead and then show that it's also `live` and images (in some simple way).

Reply:
That's also a good point. I'm implementing the confusion matrix (for Vega) in the newer example-dvc-exp-init project now. Instead of DVCLive, we can start from the basic plots using Vega-Lite, then move to images as plots. Which path would make more sense?